README.md

```diff
@@ -1,41 +1,150 @@
-# Website
+<h1 align="center">Deriv API</h1>
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+This repository contains the various platforms of the Deriv application.
```

Comment:
I don't know exactly what this line means. What platforms are we talking about? Is this perhaps better? -> `This repository contains the information and code related to the Deriv API documentation.`

Reply:
Made this change